src/main.rs

```diff
@@ -160,7 +160,8 @@ fn main() {
                 .help(
                     "Network parameters \"mac=<mac_addr>,\
                      sock=<socket_path>, num_queue_pairs=<number_of_queue_pairs>,\
-                     queue_size=<size_of_each_queue>, num_vectors=<number_of_vectors>\"",
+                     queue_size=<size_of_each_queue>, num_vectors=<number_of_vectors>,\
+                     server=<is_server>\"",
```

Comment:
I'm unclear what `server` means in this case. There are two terminology in case of vhost-user, client/server and master/slave.
And the way it usually works, we have the vhost-user daemon as the server and the slave, while the VMM is the client and the master.
Can you elaborate about the use case where the VMM is used as a server?

Reply:
Server here means the server in client/server. This option is added under considering of two points. On one hand, OVS has vhostuserclientport, add server mode for VMM to connect such type of port. On the other hand, I'm considering about the socket reconnection support. Although it is not actually added here, but to have VMM in server mode seems to be an easy way to fix reconnection issue.